### PR TITLE
feat: implement vue template

### PR DESCRIPTION
**Pull request by samobasquiat**

### CODE REVIEW

File: vue/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@rango-dev/widget-embedded": "^0.25.0",
```
Comment (RyukTheCoder) [inline vue/package.json:12]: In other examples, widget version is set to 'latest'

File: vue/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Rango Widget- Vue Template</title>
```
Comment (RyukTheCoder) [inline vue/index.html:7]:
```suggestion
    <title>Rango Widget - Vue Template</title>
```

File: vue/.vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (RyukTheCoder) [inline vue/.vscode/extensions.json:1]: Is this file required?

File: vue/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "react": "^18.2.0",
```
Comment (RyukTheCoder) [inline vue/package.json:13]: Can you explain why 'react' and 'react-dom' should still be included in 'dependencies'?

**Review by RyukTheCoder — CHANGES_REQUESTED** [review]

File: vue/package.json (hunk 4 above)
Comment (samobasquiat) [reply vue/package.json:13]: Because the new package we use (`veaury`) does not have `react` & `react-dom` itself and only has them as peer dependencies. As a result, they need to be installed as well, like in this example:
https://github.com/devilwjp/veaury/blob/master/dev-project-vue3/package.json

**Review by RyukTheCoder — APPROVED** [review]
